src/cluster_legacy.c

```diff
@@ -3463,6 +3568,15 @@ void clusterLinkConnectHandler(connection *conn) {
     serverLog(LL_DEBUG, "Connecting with Node %.40s at %s:%d", node->name, node->ip, node->cport);
 }
 
+static inline int isSigAndMsgLenValid(clusterMsg *hdr) {
```

Comment:
I'd suggest the following to help with readability.
```suggestion
static inline int isClusterMsgSignatureAndLengthValid(clusterMsg *hdr) {
```

Reply:
This looks like personal preference where we should leave some freedom to the author IMO. Sig and len are the names of the struct fields and I can see the benefit in having them match. It's a small macro-like function to encapsulate some accesses to these fields. @PingXie do you want to rename the struct fields too?

Reply:
Sig is not a common abbreviation and it took me a while to realize that it is short for "signature" as opposed to "signal". Let's at least fix the abbreviation?

Reply:
Sure, I don't mind. Shall we rename the field in the struct too? There is some value in having them match imo.

Reply:
Actually I don't mind your suggestion. I just think it's a litle too picky. The name sig refers to the struct field with the same name, which is named like that since forever.

Reply:
Now I see what you mean. Before the intoduction of this function, the code is

                /* Perform some sanity check on the message signature
                 * and length. */
                if (memcmp(hdr->sig, "RCmb", 4) != 0 || ntohl(hdr->totlen) < CLUSTERMSG_MIN_LEN) {

Here, there'a a comment explaining signature and length. When reading this function out of context, the comment can't be seen.

We could add the comment above the function definition too, but even better is to encode this information in the function name itself. Thus, @PingXie I agree with your suggestion. :)

Reply:
Done